rocketmq-connect/rocketmq-connect-cli/src/main/java/org/apache/rocketmq/connect/tools/command/QueryConnectorConfigSubCommand.java

```diff
@@ -0,0 +1,67 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+
+package org.apache.rocketmq.connect.tools.command;
+
+import org.apache.commons.cli.CommandLine;
+import org.apache.commons.cli.Option;
+import org.apache.commons.cli.Options;
+
+import org.apache.rocketmq.connect.tools.commom.Config;
+import org.apache.rocketmq.connect.tools.utils.RestSender;
+import org.apache.rocketmq.tools.command.SubCommandException;
+
+public class QueryConnectorConfigSubCommand implements SubCommand {
+
+    private final Config config;
+
+    public QueryConnectorConfigSubCommand(Config config) {
+        this.config = config;
+    }
+
+    @Override
+    public String commandName() {
+        return "queryConnectorConfig";
+    }
+
+    @Override
+    public String commandDesc() {
+        return "Get configuration information for a connector";
+    }
+
+    @Override
+    public Options buildCommandlineOptions(Options options) {
+        Option opt = new Option("c", "connectorName", true, "connector name");
+        opt.setRequired(true);
+        options.addOption(opt);
+
+        return options;
+    }
+
+    @Override
+    public void execute(CommandLine commandLine, Options options) throws SubCommandException {
+        try {
+            String connectorName = commandLine.getOptionValue('c').trim();
+            String url = "http://" + config.getHttpAddr() + ":" + config.getHttpPort() + "/connectors/" + connectorName + "/config";

```

Comment:
we better leave no magic value in code, tho there are many other places are using magic values .... (Let's see if it is possible to try to minimize use of magic values)

Reply:
Updated. Use java.net.URL class replace magic value